libcaliptra/test/if_test/if_test.mk

```diff
@@ -0,0 +1,26 @@
+Q=@
+
+SOURCE = if_test.c if_test_impl.c
+OBJS := $(patsubst %.c,%.o, $(filter %.c,$(SOURCE)))
+
+CALIPTRA_API = ../../src/caliptra_api.o
+
+.PHONY = run clean
+
+TARGET = if_test
+
+INCLUDE = ../../inc/
+
+$(TARGET): $(OBJS)
```

Comment:
you already have hwmod_test which is regressed, can we get rid of the if_test ?

Reply:
We can.